.gitignore

```diff
@@ -22,3 +22,4 @@ diff*png
 output*png
 *.code-workspace
 Brewfile.lock.json
+CMakeLists.txt.user
```

Comment:
What is this file?

Reply:
Qt creator project specific settings

Reply:
So are you done on your side? If so I'd like to test and merge this soon as it will probably create conflicts on current and future PRs.

Reply:
> Qt creator project specific settings

But then one will ask, why Qt creator will be putting its mess here? dartkable is Gtk...

Reply:
I'm using QT creator as IDE, and it creates this stuff when using CMake project

and yes i beieve i'm done with this PR